free5gc-operator/operator/role-leader-election.yaml

```diff
@@ -1,11 +1,16 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
+
 metadata:
+  namespace: free5gc
+  name: free5gc-operator-leader-election-role
   labels:
-    app.kubernetes.io/name: free5gc-operator
+    app.kubernetes.io/name: role
```

Comment:
Yeah, pretty sure that's not what app.kubernetes.io/name is for. It should be the name of the application, like "free5gc-operator". See https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#applications-and-instances-of-applications